script/docker-build-and-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ev
+app_name="hub"
```

Comment:
Only change I'd make is adding something like https://github.com/travis-ci/travis-logs/blob/master/script/docker-build-and-push#L5-L8 here so it doesn't always build. We could also make it a conditional build step. Either way!

Reply:
I modified as you suggested, and also added an external build trigger on Docker hub. (We should discuss this at some point)

Reply:
Oh, I forgot about the private gem issue. Need to fix that still.

Reply:
> I modified as you suggested, and also added an external build trigger on Docker hub. (We should discuss this at some point)

Dope! And indeed.